learn/core-concepts/tx-fees.md

```diff
@@ -305,7 +305,7 @@ The minimum gas price for each network is as follows:
 === "Moonbase Alpha"
     |     Variable      |   Value    |
     |:-----------------:|:----------:|
-    | Minimum Gas Price | 0.125 Gwei |
+    | Minimum Gas Price | 0.03125 Gwei |
```

Comment:
Lets use the variable here and please make sure to format the table afterwards 🙂